jbpm-services/jbpm-kie-services/src/test/java/org/jbpm/kie/services/test/QueryServiceImplTest.java

```diff
@@ -1019,15 +1019,6 @@ public void testGetFilteredProcessInstances() {
         assertEquals(1, instances.size());
         assertEquals(1, (int) instances.iterator().next().getState());
 
-        // let's now change the roles so user should not see instances
-        roles.clear();
-        roles.add("employees");
-        identityProvider.setRoles(roles);
-        identityProvider.setName("anotherUser2");
-
-        instances = queryService.query(query.getName(), ProcessInstanceQueryMapper.get(), new QueryContext());
-        assertNotNull(instances);
-        assertEquals(0, instances.size());
```

Comment:
What about leaving this use case but just changing the number in the assert? So we are sure that anotherUser2 can see the instances as well?